greykite/common/data_loader.py

```diff
@@ -71,7 +71,7 @@ def get_data_home(data_dir=None, data_sub_dir=None):
             data_home = os.path.abspath(os.path.join(data_dir, data_sub_dir))
 
         if not os.path.exists(data_home):
-            raise ValueError(f"Requested data directory '{data_home}' does not exist.")
+            raise ValueError(f"Requested data directory '{data_dir}' does not exist.")
```

Comment:
Tato změna je kvůli neprocházejícím testům? Správně by zde totiž měla být `data_home` (může to být i podsložka `data_dir`), pokud někde test neprochází kvůli lomítkům ve zprávě, tak by bylo dobré kontrolu přepsat pomocí reguláru podobně, jako jsi to udělala u `test_get_df` v `greykite/tests/common/test_data_loader.py`.

Reply:
Ano, test neprochází, pro `data_sub_dir = None`, protože pak srovnává `data_dir` a `data_home` (test_data_loader.py ř.29): ` with pytest.raises(ValueError, match=f"Requested data directory '{data_dir}' does not exist."):`
Navíc v tom matchi píšou, `Requested data directory`, což jsem pochopila, jako `data_dir` na vstupu. 
Tak jsem opravila kód, aby odpovídal testu.

Ale máš pravdu, že co hledá, `if not os.path.exists(data_home):`, je `data_home`, tak by tam měl být `data_home`... jenže pak  vlastně požaduje, aby `data_dir == os.path.abspath(data_dir)`.. takže zase Windows, `'/home/data' != 'C:\\home\\data'`. Upravila jsem hlášku i test.